app/views/guests/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1> Name: <%= @guest.name %></h1>
+
+<div id="rooms_stayed">
+<p> Rooms Stayed In:</p>
+<ul>
+<% @guest.rooms.each do |room| %>
+<li> <%= room.suite %> ($<%= room.rate %> at the <%= room.hotel.name %>) </li>
+<% end %>
+</ul>
+</div>
+
+<div id="add_rooms">
+	<%= form_with url: "/guests/#{@guest.id}", method: :patch, local: true do |form| %>
```

Comment:
I think the first part of this path is ok, but i disagree that we're doing an `update` functionality here, we're really creating a new relationship between an existing Room and an existing Guest. 

We'll need the `guest_id` in the path or in the request body of the form, but the URL could also be: 
```
/guests/:guest_id/rooms
```
and then we send in the `room_id` in the form that way. Also, the method should be `:post` since we're creating a relationship, not updating either Guest or Room itself.

Reply:
The path could also potentially be
```
/room_guests
```
with a `:post` method. If you did that, you'd have to include a `form.hidden_field` whose value is the `guest_id`.